configserver/src/test/java/com/yahoo/vespa/config/server/ModelContextImplTest.java

```diff
@@ -89,11 +88,11 @@ public void testModelContextTest() {
         assertEquals(Optional.empty(), context.wantedDockerImageRepo());
         assertEquals(new Version(7), context.modelVespaVersion());
         assertEquals(new Version(8), context.wantedNodeVespaVersion());
-        assertEquals(1.0, context.properties().defaultTermwiseLimit(), 0.0);
-        assertFalse(context.properties().useAsyncMessageHandlingOnSchedule());
-        assertEquals(0, context.properties().contentNodeBucketDBStripeBits());
-        assertEquals(0x400000, context.properties().mergeChunkSize());
-        assertEquals(0.5, context.properties().feedConcurrency(), 0.0);
+        assertEquals(1.0, context.properties().featureFlags().defaultTermwiseLimit(), 0.0);
```

Comment:
perhaps featureFlags() should be directly on the context?

Reply:
That was my original plan, but it turned to out be too clunky to implement.